bin/build_tasks.sh

```diff
@@ -18,7 +18,7 @@ fi
 cd ../../
 
 failure=0
-for TASK in $(ls -d tasks/* | egrep -v "package|dr_dbutils|pg_utils|shared_libraries")
+for TASK in $(ls -d tasks/* | egrep -v "package|dr_dbutils|pg_utils|shared_libraries|inventory_report")
```

Comment:
I am assuming this is temporary until the lambda is created in a different sprint correct?